admin/sql/create_tables.sql

```diff
@@ -95,7 +95,8 @@ CREATE TABLE "user" (
     created        TIMESTAMP   NOT NULL,
     musicbrainz_id VARCHAR,
     show_gravatar  BOOLEAN     NOT NULL DEFAULT False,
-    is_blocked     BOOLEAN     NOT NULL DEFAULT False
+    is_blocked     BOOLEAN     NOT NULL DEFAULT False,
+    license_choice VARCHAR     NOT NULL
```

Comment:
This should be `NULLABLE` since new users would not have chosen a default license and none of the existing users will have a default license.